.pre-commit-config.yaml

```diff
@@ -5,3 +5,16 @@
     - id: git-check  # Configure in .gitattributes
     - id: git-dirty  # Configure in .gitignore
     - id: shellcheck
+- repo: https://github.com/pre-commit/mirrors-mypy
+  sha: v0.701
+  hooks:
+    - id: mypy
+- repo: https://github.com/pre-commit/mirrors-isort
+  sha: v4.3.20
+  hooks:
+    - id: isort
+- repo: https://github.com/pre-commit/pre-commit-hooks/
+  sha: v2.2.3
+  hooks:
+    - id: flake8
+    - id: check-ast
```

Comment:
if you've got flake8 there's no need for `check-ast`, `check-ast` is mostly only useful if you need to support multiple versions of python but haven't yet made them flake8 compatible with the other versions (basically as a transition period for porting)